invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/deposits/RDMDepositForm/RDMDepositForm.js

```diff
@@ -250,160 +251,170 @@ export class RDMDepositForm extends Component {
     };
   }
 
+  contextRef = createRef()
```

Comment:
According to [this](https://stackoverflow.com/questions/54620698/whats-the-difference-between-useref-and-createref) I would put the `contextRef` initialization in the constructor to avoid recreating it in every render. Althoug h, `RDMDepositForm` component shouldn't rerender, it doesn't depend on dynamic state, I would move it to the constructor, just to be safe :) I will merge this and I can include it in one of my upcoming PRs!